index.html

```diff
@@ -0,0 +1,208 @@
+            </ul>
+          </div>
+          <div class="project">See Project</div>
+        </div>
+      </div>
+
+      <div class="card">
+        <img class="img" src="image/card-3.svg" alt="">
+        <div class="content">
+          <div class="topic">Tonic</div>
+          <div class="sub-topic">
+            <span>CANOPY</span>
+            <span class="dot"></span>
+
+            <span>Back End Dev</span>
+            <span class="dot"></span>
+
+            <span>2015</span>
+          </div>
+          <div class="description">
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+          </div>
+          <div class="button-links">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+          </div>
+          <div class="project">See Project</div>
+        </div>
+      </div>
+
+      <div class="card">
+        <img class="img" src="image/card-4.svg" alt="">
+        <div class="content">
+          <div class="topic">Multi-Post <br /> Stories</div>
+          <div class="sub-topic">
+            <span>CANOPY</span>
+            <span class="dot"></span>
+
+            <span>Back End Dev</span>
+            <span class="dot"></span>
+
+            <span>2015</span>
+          </div>
+          <div class="description">
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+          </div>
+          <div class="button-links">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+          </div>
+          <div class="project">See Project</div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div>
+    <div class="topic">About Myself</div>
+    <div class="description">
+      Hello I'm a software developer! I can help you build a product ,
+      feature or website Look through some of my work and experience!
+      If you like what you see and have a project you need coded, don't
+      hestiate to contact me.
+    </div>
+    <div class="menuSection">
+      <a href="#" target="_blank">LET'S CONNECT</a>
+      <div class="socialMedia">
+        <img src="image/Twitter.png" alt="" />
+        <img src="image/Linkedin.png" alt="" />
+        <img src="image/Meduim.png" alt="" />
+        <img src="image/Github.png" alt="" />
+        <img src="image/angellist.png" alt="" />
+      </div>
+    </div>
+    <div class="project">Get my resume</div>
+
+    <ul>
+      <li>
+        <div class="topic item">Languages</div>
+        <ul class="languages">
+          <li> <img src="image/js.svg" alt=""> JavaScript</li>
+          <li> <img src="image/html5.svg" alt=""> HTML</li>
+          <li> <img src="image/css3.svg" alt=""> CSS</li>
+        </ul>
+      </li>
+      <li class="topic item">Frameworks</li>
+      <li class="topic item">Skills</li>
```

Comment:
- [ ] Kindly adjust the font size and weight of the text to match the [Figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) chosen 👍. You can check out the font size and weight from the description on Figma

| Figma Design  | Your Design |
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/63473584/177019271-19e2bd57-bc96-4153-be50-0a94cae6a341.png) | ![image](https://user-images.githubusercontent.com/63473584/177019281-fd97a7b0-7a00-47d7-a2ed-8c50211e3745.png) |